### PR TITLE
add support criteria (title) parser

### PR DESCRIPTION
**Pull request by OriHoch**
* related issue: #55

### CODE REVIEW

File: tests/pipelines/support/test_support_criteria_parser.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                    # the expected data from the parser
+                    {"name": "expected_purpose", "type": "string"}
+                ]
+            }
+        }]
+    })
+    i = 0
+    num_parsed = 0
+    num_expected_purposes = 0
+    for i, row in enumerate(dp.resources[0].iter()):
```
Comment (akariv) [inline tests/pipelines/support/test_support_criteria_parser.py]: should be `iter(keyed=True)`
(this was a breaking change in the `datapackage` library 🙁 )
will also fix the failing test I believe.

**Comment by akariv** [comment]
Hey, can you take a look and see why the test is failing?

**Comment by OriHoch** [comment]
I guess pre-v1 it returned empty string and post v1 it returns None ...
I'll fix it

**Comment by akariv** [comment]
Yes, a common phenomenon with the version update.

**Comment by OriHoch** [comment]
fixed

**Comment by akariv** [comment]
I thought this was already merged? @OriHoch

**Comment by OriHoch** [comment]
@akariv no, it doesn't look merged to me..

**Comment by akariv** [comment]
Okay, so can you please rebase?

**Comment by OriHoch** [comment]
done

**Pull request merged**